app/components/message_attachments/action_menu/index.js

```diff
@@ -4,23 +4,23 @@
 import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
-import {doPostAction} from 'mattermost-redux/actions/posts';
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
 
-import {setMenuActionSelector} from 'app/actions/views/post';
+import {setMenuActionSelector, selectAttachmentMenuAction} from 'app/actions/views/post';
 
 import ActionMenu from './action_menu';
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
     return {
         theme: getTheme(state),
+        selected: state.views.post.submittedMenuActions[ownProps.postId],
```

Comment:
a selector would be good here

Reply:
Do we really need a selector for something so simple and specific? Seems like extra code for no benefit

Reply:
no not really is just for readability, but 0/5 if you want to change it or not